src/quo/components/wallet/address_text/view.cljs

```diff
@@ -0,0 +1,26 @@
+(ns quo.components.wallet.address-text.view
+  (:require [quo.components.markdown.text :as text]
+            [quo.foundations.colors :as colors]
+            [utils.address :as utils]))
+
+(defn- view-internal
+  [{:keys [networks address type blur? theme]}]
+  [text/text
+   (map (fn [network]
+          ^{:key (str network)}
+          [text/text
+           {:size  :paragraph-2
+            :style {:color (colors/resolve-color network theme)}}
+           (str (subs (name network) 0 3) (when (= network :arbitrum) "1") ":")])
```

Comment:
can we remove this logic from here? I have this addressed in a separate pr but it would be better to just follow the approach I have taken and leave the handing of this to status-im code..
see
https://github.com/status-im/status-mobile/pull/17650/files
file:  src/status_im2/contexts/wallet/account/tabs/about/view.cljs

Reply:
@J-Son89 I can update the logic here. But why put in in `status-im` namespace. This component is repeated multiple times in `quo`

Reply:
Yea and it should not be handled in the quo components- it should just be passed in as a detail and the quo component renders it without processing. Eg we have a short name prop in the network property and it uses that here

We handle this once by doing it in status-im as this data is centralised anyway.

Reply:
why does this component need to have knowledge about how we handle the short names of the wallet networks? Particularly this arb1 logic. These short network names are consistent across the designs/views/components.
In my other pr i centralise this data and we have short-names defined for the network details.

Reply:
I agree with @J-Son89 here, the reasoning behind this is because quo components should be as dumb as possible, arbitrum logic belongs to the wallet domain and this component should be abstracted from that logic.

Reply:
ok, thanks for clearing this up :+1: 
Also thanks @briansztamfater  :smile: